src/components/user.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export const User = (({ users }) =>
+  users.map(
+    (user) => (
+      <div key={user.id}>
+        <h2>{user.login}</h2>
+        <img src={user.avatar_url} alt='' height={100}></img>
+        <p>{user.organizations_url}</p>
+        <p>{user.received_events_url}</p>
```

Comment:
JS convention for variable/function naming is `camelCase`. Whereas from the APIs, we always receive data formatted in `snake_case` variables. So one way to handle this is when you save the data inside redux store, first process it to change cases via some library such as `change-case-object`, and then save it into store. So that when you use it from the store, it will be in standard `camelCase` format that we should be using in JS.